### PR TITLE
First draft of IPFS Weekly #53

### PR DESCRIPTION
**Pull request by jennwrites**
Looking for feedback and suggestions on this draft by Monday, August 5 at 12pm Central so that we can send it out via TinyLetter (and post on the blog) Tuesday, August 6. 🌞

Thanks!

### CODE REVIEW

**Comment by ianopolous** [comment]
Under IPFS in the wild could we please add:

[Peergos - fast seeking and encrypted history](https://peergos.org/blog#fast_seeking_and_encrypted_history_), 1 August 2019

and under new releases could we please add:

[Peergos v0.1.0](https://alpha.peergos.net/public/peergos/releases/v0.1.0/) - Fast seeking, encrypted history, and space usage display

Many thanks!

**Comment by Leela-ann** [comment]
Under IPFS in the wild:
SF [IPFS Meet & Hack](https://www.meetup.com/San-Francisco-IPFS/events/261811887/) August 8, 2019

thank you!

**Comment by jennwrites** [comment]
> Under IPFS in the wild:
> SF [IPFS Meet & Hack](https://www.meetup.com/San-Francisco-IPFS/events/261811887/) August 8, 2019
> 
> thank you!

Already included under community events. 👍

**Comment by Leela-ann** [comment]
Thank you!

File: published/053-2019-august-6.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+## The latest
+
+### IPFS Project Q3 Priorities & Working Groups
+https://ipfs.io/blog/2019-07-31-operation-task-force/
+
+### Multiple-Choice Lessons in ProtoSchool
+Did you know that not every lesson in ProtoSchool includes a coding challenge? ProtoSchool has long supported both coding exercises and text-based lessons, but today we're introducing [a new lesson type: multiple-choice](https://twitter.com/ProtoSchool/status/1156942749595312128). 🎉
+
+As the author of a tutorial, you can now present the learner with a multiple-choice quiz after she reads the text of your lesson, adding some interactivity and a chance to check comprehension before moving on to the next topic.
```
Comment (terichadbourne) [inline published/053-2019-august-6.md]:
```suggestion
As the author of a tutorial, you can now present the learner with a multiple-choice quiz after she reads the text of your lesson, adding some interactivity and a chance to check comprehension before moving on to the next topic. Learn more about [building ProtoSchool tutorials](https://proto.school/#/build).
```

File: published/053-2019-august-6.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
++ **14 August 2019:** Join [Women Who Code Cloud](https://www.womenwhocode.com/cloud/events) for an [Introduction to IPFS](https://zoom.us/webinar/register/WN_jnKnkxjJR3OOxf3kPa7Xfg) inspiring women to excel in technology careers. Remote event.
++ **21-23 August 2019:** [DappCon Berlin](https://www.dappcon.io/): A global developer conference for Ethereum Dapps, Infrastructure, and token engineering, Berlin Germany.
++ **23-25 August 2019:** [ETHBerlinZwei](https://ethberlinzwei.com/) is the second iteration of ETHBerlin, a hackathon, a culture festival, an educational event, a platform for hacktivism, and a community initiative to push the decentralized ecosystem forward. Berlin, Germany.
++ **RESCHEDULED – 27-30 September 2019:** [Offline Camp Oregon](http://offlinefirst.org/camp/reschedule) had to be postoned, but it’s still happening! Here’s past campers’ take on our unique unconference retreat.
```
Comment (terichadbourne) [inline published/053-2019-august-6.md]:
```suggestion
+ **RESCHEDULED – 27-30 September 2019:** [Offline Camp Oregon](http://offlinefirst.org/camp) had to be postponed, but it’s still happening! [Hear how past campers describe this unique unconference retreat](https://youtu.be/FNtpPW_7H1k) focused on the [Offline First](http://offlinefirst.org/) approach to web development and design. 
```

**Review by terichadbourne — CHANGES_REQUESTED** [review]
Thanks for the ProtoSchool and Offline Camp callouts, @renrutnnej! Added a couple of suggested tweaks/links.

**Comment by jennwrites** [comment]
@moyid Draft is ready for translation. Thank you 🙏

**Comment by ianopolous** [comment]
@renrutnnej Just checking if you saw my comment above? None of it seems to be included.

**Comment by jennwrites** [comment]
Apologies if I missed your item, I’m working under the weather today trying
to get the newsletter out on its regular schedule.

I will be sure to include in next issue.

**Comment by ianopolous** [comment]
@renrutnnej I put in it a 4 days ago specifically to make this mail out. We care a lot about these. What do I need to do ensure inclusion? Can I submit a PR now with it included?

**Comment by ianopolous** [comment]
@renrutnnej I've made a pr into this branch with my contributions:
https://github.com/ipfs/newsletter/pull/109

**Comment by ianopolous** [comment]
@momack2 Is there anything we can do about this?

**Comment by jennwrites** [comment]
@ianopolous I added your changes this morning. Thank you so much for your patience and support of the newsletter.

**Comment by jennwrites** [comment]
@moyid Added minor changes to lines 28 && 53 for translation.

**Comment by moyid** [comment]
@renrutnnej  ack. I will push my translation to the blog repo after this is published

**Comment by jennwrites** [comment]
IPFS Weekly #53 was published

- To the [IPFS blog](https://blog.ipfs.io/weekly-53/)
- Sent to 2864 subscribers [via Tinyletter](https://tinyletter.com/ipfsnewsletter/letters/ipfs-weekly-53)

**Pull request merged**